lib/jnpr/junos/utils/fs.py

```diff
@@ -264,6 +265,35 @@ def _decode(fs):
         return dict((_name(fs), _decode(fs)) for fs in rsp.xpath('filesystem'))
 
     # -------------------------------------------------------------------------
+    # directory_usage - filesystem directory usage
+    # -------------------------------------------------------------------------
+
+    def directory_usage(self, path="."):
+        """
+        Returns the directory usage, similar to the unix "du" command.
+
+        :returns: approximate directory usage, including subdirectories, in bytes
+        """
+        rsp = self._dev.rpc.get_directory_usage_information(path=path, depth="0")
+
+        used_space = rsp.findtext("directory/used-space")
+
+        if used_space is None:
+            return None
+
+        used_space = used_space.strip()
+
+        multiplier = {
```

Comment:
Calculate the `bytes` value using the number of blocks (the `used-blocks` attribute of the `<used-space>` element. Then it's simply: `bytes = blocks * BLOCK_SIZE`. You can define `BLOCK_SIZE = 512` at the beginning of the method.